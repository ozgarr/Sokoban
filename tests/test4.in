---
@z-
-A-

z4
z8
z6
z2
0
0
a8
a4
z2
0
z4
z2
a6
0
0
a6
a6
z4
0
0
0
0
.
