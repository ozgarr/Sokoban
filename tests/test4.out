---
@z-
-A-
---
z@-
-A-
---
z@-
-A-
---
z@-
-A-
---
@--
zA-
---
z@-
-A-
---
@z-
-A-
---
@z-
-A-
---
-z-
a*-
---
-@-
aZ-
---
-z-
a*-
---
z@-
a+-
---
z@-
a+-
---
z@-
a+-
---
-z-
a*-
---
@z-
-A-
---
-z-
-*a
---
-z-
-*a
---
z@-
-+a
---
-z-
-*a
---
@z-
-A-
---
@z-
-A-
---
@z-
-A-
